README.md

```diff
@@ -21,48 +21,225 @@ data to New Relic as either logs or events. The exporter currently supports
+
+### On-host
+
+To use the Salesforce Exporter on a host, perform the following steps.
 
 1. Clone this repository
 1. Run `pip install -r requirements.txt` to install dependencies
 
+Once installed, the Salesforce Exporter can be run from the repository root
+using the command `python src/__main__.py`. See the section
+[Command Line Options](#command-line-options) and [Configuration](#configuration)
+for more details on using the exporter.
+
 ### Docker
 
-1. Clone this repository
-1. Run `docker build -t nr-sf-eventlogs .` to build the docker image.
+A Docker image for the Salesforce Exporter is available at
+[https://hub.docker.com/r/newrelic/newrelic-salesforce-exporter](https://hub.docker.com/r/newrelic/newrelic-salesforce-exporter). This image can be used in one of two ways.
+
+#### Run directly from [DockerHub](https://hub.docker.com/)
+
+The Salesforce Exporter [Docker image](https://hub.docker.com/r/newrelic/newrelic-salesforce-exporter)
+can be run directly from [DockerHub](https://hub.docker.com/). To do this, the
+[`config.yml`](#configyml) must be mapped into the running container. It can
+be mapped using the default filename or using a custom filename. In the case of
+the latter, the `-f` [command line option](#command-line-options) must be
+specified with the custom filename. One of the same methods can be used to map
+an [event type fields mapping file](#event-type-fields-mapping-file) and/or a
+[numeric fields mapping file](#numeric-fields-mapping-file) into the running
+container. In addition, environment variables can be passed to the container
+using `docker run` with [the `-e`, `--env`, or `--env-file` options](https://docs.docker.com/reference/cli/docker/container/run/#env)
+for [configuration parameters](#configuration) that can be specified via
+environment variables. See below for examples.
+
+**Example 1: Using the default configuration filename**
+
+In the following example, the file `config.yml` in the current directory on the
+host system is mapped with the default configuration filename in the container
+(`config.yml`). In addition, the [`license_key`](#license_key) value is
+specified using the `NR_LICENSE_KEY` environment variable and the
+[application name](https://docs.newrelic.com/docs/apm/agents/python-agent/configuration/python-agent-configuration/#app_name)
+and [license key](https://docs.newrelic.com/docs/apm/agents/python-agent/configuration/python-agent-configuration/#license_key)
+agent parameters for the built-in [New Relic Python agent](#new-relic-python-agent)
+are specified using the `NEW_RELIC_APP_NAME` AND `NEW_RELIC_LICENSE_KEY`
+environment variables, respectively. No command line argument are passed to the
+exporter.
 
-## Usage
+```bash
+docker run -t --rm --name salesforce-exporter \
+   -v "$PWD/config.yml":/usr/src/app/config.yml \
+   -e NR_LICENSE_KEY=$NR_LICENSE_KEY \
+   -e NEW_RELIC_APP_NAME="New Relic Salesforce Exporter" \
+   -e NEW_RELIC_LICENSE_KEY=$NEW_RELIC_LICENSE_KEY \
```

Comment:
Maybe I just haven't noticed it before, but having 2 `*_LICENSE_KEY` vars is confusing. Can we consolidate those?

Reply:
It _is_ confusing.  And irritating.  And I think you did notice it before.  We _can_ consolidate them but `NR_LICENSE_KEY` was used historically before we ever added the Python APM agent in so I left both for "backwards compatibility".  If we want to remove them moving forward, could you open an issue for it?